hooks/hookexecution/mocks_test.go

```diff
@@ -287,3 +315,26 @@ func (e mockUpdateBidderResponseHook) HandleRawBidderResponseHook(_ context.Cont
 
 	return hookstage.HookResult[hookstage.RawBidderResponsePayload]{ChangeSet: c}, nil
 }
+
+func (e mockFailedMutationHook) HandleAllProcessedBidResponsesHook(_ context.Context, _ hookstage.ModuleInvocationContext, _ hookstage.AllProcessedBidResponsesPayload) (hookstage.HookResult[hookstage.AllProcessedBidResponsesPayload], error) {
+	changeSet := &hookstage.ChangeSet[hookstage.AllProcessedBidResponsesPayload]{}
+	changeSet.AddMutation(func(payload hookstage.AllProcessedBidResponsesPayload) (hookstage.AllProcessedBidResponsesPayload, error) {
+		return payload, errors.New("key not found")
+	}, hookstage.MutationUpdate, "some-bidder", "bids[0]", "deal_priority")
+
+	return hookstage.HookResult[hookstage.AllProcessedBidResponsesPayload]{ChangeSet: changeSet}, nil
+}
+
```

Comment:
Nitpick: can you move this up so it is near the other `mockFailedMutationHook` functions just above line 305 `type mockUpdateBidderResponseHook struct{}`?